requirements.txt

```diff
@@ -2,3 +2,7 @@ requests
 flask
 cryptography
 bump2version
+click
+prompt_toolkit
+threading
```

Comment:
Do not use threading for an asynchronous program. Utilise the async/await architecture instead. Please remediate.